.github/workflows/test.yml

```
@@ -43,3 +41,26 @@ jobs:
         pip install maturin pytest
         maturin develop
         pytest tests
+build-macos:
```

I think that you are missing indentation, which is the reason why the action fails currently: https://github.com/avast/yari/actions/runs/7396538047
```suggestion
  build-macos:
```

---

Oops!